.github/labeler.yml

```
@@ -0,0 +1,2 @@
+'squad: system':
+  - ./**/*
```

```suggestion
  - **/*
```

Maybe just this for matching everything? Not sure though

---

I tried that in first commit. The Action failed. Almost like the global glob needed to be grounded in the current directory. https://github.com/carbon-design-system/carbon/pull/4892/commits/a06394d864627c3bf08d5986e79fdcdc485c65f5